example/components/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+---
+version: '2'
+
+services:
+  server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "5000:5000"
```

Comment:
Should use port 5006+ (or something around there) just to make the integration testing easier

Reply:
Agreed. Even I ran into the problem with other things I had running.